lib/SILGen/SILGenExpr.cpp

```diff
@@ -2987,8 +2986,8 @@ getOrCreateKeyPathEqualsAndHash(SILGenModule &SGM,
 
   auto indexLoweredTy = SGM.Types.getLoweredType(indexTupleTy);
   // Get or create the equals witness
-  [&unsafeRawPointerTy, &boolTy, &genericSig, &C, &indexTypes, &equals, &loc,
-   &SGM, &genericEnv, &indexLoweredTy, &indexes]{
+  [unsafeRawPointerTy, boolTy, genericSig, &C, &indexTypes, &equals, loc,
+   &SGM, genericEnv, expansion, indexLoweredTy, indexes]{
```

Comment:
...why does this even bother with the explicit captures?

Reply:
I don’t know, it was like this when I got here, man :)

Reply:
It was a bug-hunting exercise against referencing some things that weren't supposed to be referenced in this code block.